python/pyspark/sql/functions.py

```diff
@@ -2927,7 +2927,7 @@ def pandas_udf(f=None, returnType=None, functionType=None):
 __all__ = [k for k, v in globals().items()
            if not k.startswith('_') and k[0].islower() and callable(v) and k not in blacklist]
 __all__.sort()
-
+__all__ += ["PandasUDFType"]
```

Comment:
@kevinyu98 should this be before the sort() ?

Reply:
yah, let me change it. Thanks